src/Lachain.Core/CLI/ConsoleCommands.cs

```diff
@@ -188,7 +189,7 @@ public string SendContract(string[] arguments)
                 Money.Zero,
                 methodSignature,
                 ContractEncoder.RestoreTypesFromStrings(arguments.Skip(3)));
-            var signedTx = _transactionSigner.Sign(tx, _keyPair);
+            var signedTx = _transactionSigner.Sign(tx, _keyPair, HardforkHeights.IsHardfork_8Active(_blockManager.GetHeight()));
```

Comment:
I think it should be _blockManager.GetHeight() + 1, if it goes to the next block

Reply:
resolved